lib/Widgets/DynamicNotebook.vala

```diff
@@ -1197,7 +1215,14 @@ namespace Granite.Widgets {
         }
 
         private void on_tab_closed (Tab tab) {
-            if (Signal.has_handler_pending (this, Signal.lookup ("close-tab-requested", typeof (DynamicNotebook)), 0, true)) {
+            if (
+                Signal.has_handler_pending (
+                    this,
+                    Signal.lookup ("close-tab-requested", typeof (DynamicNotebook)),
+                    0,
+                    true
+                )
+            ) {
```

Comment:
This block is pretty ugly, idk the best way to nest/wrap it

Reply:
You can go with two level indentation here.
```
 if (Signal.has_handler_pending (
        this,
        Signal.lookup ("close-tab-requested", typeof (DynamicNotebook)),
        0,
        true)
    ) {
```